manager/src/main/resources/org/openremote/manager/setup/database/V20220404_01__Schema.sql

```diff
@@ -0,0 +1,12 @@
+create table DASHBOARD (
```

Comment:
Update this filename so it comes after other migration scripts (new one has been added since you created this)